packages/ts/src/rules/asyncPromiseExecutors.test.ts

```diff
@@ -0,0 +1,84 @@
+import rule from "./asyncPromiseExecutors.js";
+import { ruleTester } from "./ruleTester.js";
+
+ruleTester.describe(rule, {
+	invalid: [
+		{
+			code: `
+new Promise(async (resolve, reject) => {
+	resolve(42);
+});
+`,
+			snapshot: `
+new Promise(async (resolve, reject) => {
+            ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
This is a lot of squiggly range. Change the rule report range to just be the `async` keyword.

```suggestion
new Promise(async (resolve, reject) => {
            ~~~~~
```